openpype/hosts/houdini/plugins/load/load_fbx.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+"""Fbx Loader for houdini.
+
+It's almost a copy of
+'load_bgeo.py'and 'load_alembic.py'
+however this one includes extra comments for demonstration.
+
+This plugin is part of publish process guide.
+"""
+import os
+
+from openpype.pipeline import (
+    load,
+    get_representation_path,
+)
+from openpype.hosts.houdini.api import pipeline
+
+
+class FbxLoader(load.LoaderPlugin):
+    """Load fbx files to Houdini."""
+
+    label = "Load FBX"
+    families = ["staticMesh", "fbx"]
+    representations = ["fbx"]
+
+    # Usually you will use these value as default
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name=None, namespace=None, data=None):
+
+        # get file path
+        file_path = self.get_file_path(context)
+
+        # get necessary data
+        namespace, node_name = self.get_node_name(context, name, namespace)
+
+        # create load tree
+        nodes = self.create_load_node_tree(file_path, node_name, name)
+
+        self[:] = nodes
+
+        # Call containerise function which does some
+        # automations for you
+        containerised_nodes = self.get_containerised_nodes(
+            nodes, context, node_name, namespace
+        )
+
+        return containerised_nodes
+
+    def update(self, container, representation):
+
+        node = container["node"]
+        try:
+            file_node = next(
+                n for n in node.children() if n.type().name() == "file"
+            )
+        except StopIteration:
+            self.log.error("Could not find node of type `file`")
+            return
+
+        # Update the file path
+        file_path = get_representation_path(representation)
```

Comment:
Unlike `load()` which uses `self.get_file_path` this does not do `os.normpath` and replacing of backslashes to forward slashes. Is there any way we can do the same call here using the context?